packages/flutter_test/lib/src/window.dart

```diff
@@ -610,16 +619,36 @@ class FakeAccessibilityFeatures implements ui.AccessibilityFeatures {
 ///     testing and mocking purposes.
 class TestPlatformDispatcher implements ui.PlatformDispatcher {
   /// Constructs a [TestPlatformDispatcher] that defers all behavior to the given
-  /// [dart:ui.PlatformDispatcher] unless explicitly overridden for test purposes.
+  /// [ui.PlatformDispatcher] unless explicitly overridden for test purposes.
   TestPlatformDispatcher({
     required ui.PlatformDispatcher platformDispatcher,
-  }) : _platformDispatcher = platformDispatcher;
+  }) : _platformDispatcher = platformDispatcher {
+    _configuration = _platformDispatcher.configuration.copyWith();
```

Comment:
Why do we need to freeze the configuration by copying it?

Reply:
Configurations removed per other convos and issues.